src/main/resources/mustache-templates/layout/footer.html

```diff
@@ -1,19 +1,37 @@
+
+<footer class="page-footer font-small mdb-color lighten-3 pt-4">
 	<div class="container">
-		<footer>
-			<p class="pull-right"><small>HPO on FHIR</small></p>
-		</footer>
+		<div class="row">
+			<div class="col-sm-6 footer-copyright py-3">
+				© 2018: <a href="https://github.com/OCTRI/poc-hpo-on-fhir"> HPO
+					on FHIR</a>
+			</div>
+			<div class="col-sm-6">
+				<div class="float-right">
+					<a href="http://www.ohsu.edu/xd/research/centers-institutes/octri/">
+						<img src="/images/octri.gif" height="30" class="d-inline-block"
+						alt="OCTRI logo">
+					</a> <a href="https://www.jax.org"> <img src="/images/jax.png"
+						height="30" class="d-inline-block" alt="JAX logo">
+					</a>
+				</div>
+			</div>
+		</div>
 	</div>
+</footer>
+
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/jquery/1.12.4/jquery.min.js"></script>
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/bootstrap/4.0.0/js/bootstrap.min.js"></script>
```

Comment:
If you include the webjars locator dependency in your pom you can leave off the version numbers here. That makes it easier to upgrade dependencies later.

Reply:
I tried that when I first created the app. It didn't work as expected, so I kicked it down the road. I'll make a note to address it though.